### PR TITLE
Add changelog for #32956 [ci skip]

### PR DESCRIPTION
**Pull request by bogdanvlviv**
Add mention about default value of `config.active_model.i18n_full_message`.

### CODE REVIEW

File: guides/source/configuring.md [hunk 1]
```diff
@@ -307,8 +307,7 @@ All these configuration options are delegated to the `I18n` library.
 
 ### Configuring Active Model
 
-* `config.active_model.i18n_full_message` is a boolean value which controls whether the `full_message` error format can be overridden at the attribute or model level in the locale files.
-
+* `config.active_model.i18n_full_message` is a boolean value which controls whether the `full_message` error format can be overridden at the attribute or model level in the locale files. This is `false` by default.
```
Comment (kamipo) [inline guides/source/configuring.md]: Why deleted a blank line?

Comment (bogdanvlviv) [reply guides/source/configuring.md]: Not intentionally. I've returned it back.

File: actionview/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+*   Add `config.active_model.i18n_full_message` in order to control whether
```
Comment (kamipo) [inline actionview/CHANGELOG.md]: Should be in `activemodel/CHANGELOG.md`.

Comment (bogdanvlviv) [reply actionview/CHANGELOG.md]: Yes. It is already there.

**Pull request merged**